### PR TITLE
psutil package fix for os.getloadavg() crash on windows.

### PR DESCRIPTION
**Pull request by sigmantium**
Rectify issue on windows found in #283 
-Import psutil package and update os.getloadavg() for windows support.

### CODE REVIEW

**Review by amor71 — CHANGES_REQUESTED** [review]
if you make such a change: 
1. need to make sure package is available in the requirements 
2. add test coverage in the tests/ folder

Happy to add you as a collaborator and give credit once this is fully release ready and tested.

**Comment by sigmantium** [comment]
Further comment to this, with the way the psutil.getloadavg() function works on Windows, and the way that it is called within this project it may actually be redundant. The first call of the function on a windows PC will always return 0.0 as it spawns a background monitoring process and there is no history for it to compare against, see :[psutil docs](https://psutil.readthedocs.io/en/latest/index.html?highlight=getloadavg#psutil.getloadavg) to understand further.
I completely understand if its worth dropping this PR and looking for an alternative method. At the moment however, it does prevent a failure on Windows PC's.

**Comment by amor71** [comment]
@sigmantium I was reading further and I agree with your suggestion. However, would you be so kind and alter it, so it will properly work on Windows? As you correctly indicated, the first call would always return 0. and it seems like need to wait a few seconds? Can you please update the code, test, and post the results on Windows? I'm mostly running on Mac and Linux.

**Comment by sigmantium** [comment]
Changed the call to a combination of cpu_percent() and cpu_count() which is a closer implementation to the original. Has been tested on both Linux and Windows with consistent results across both. cpu_percent() is blocking, using None as the interval will show performance levels since the last time the call was made..

File: liualgotrader/requirements/release.txt [hunk 1]
```diff
@@ -23,5 +23,5 @@ tqdm==4.62.3
 toml==0.10.2
 fire==0.4.0
 opentelemetry-exporter-gcp-trace==1.0.0
-pandas_market_calendars
```
Comment (amor71) [inline liualgotrader/requirements/release.txt:26]: w need pandas_market_calendars

**Review by amor71 — CHANGES_REQUESTED** [review]
It's pretty cool! once you fix the release.txt change, I will merge, run tests on MacOS & Debian, and will release. Happy to give you credit for this, nicely done.

File: liualgotrader/requirements/release.txt (hunk 1 above)
Comment (sigmantium) [reply liualgotrader/requirements/release.txt:26]: Should be updated.

**Pull request merged**